src/torrent/tracker_list.cc

```diff
@@ -84,14 +84,14 @@ TrackerList::has_active_not_scrape_in_group(uint32_t group) const {
   return std::find_if(begin_group(group), end_group(group), std::mem_fun(&Tracker::is_busy_not_scrape)) != end_group(group);
 }
 
-// Need a custom predicate because the is_usable function is virtual.
-struct tracker_usable_t : public std::unary_function<TrackerList::value_type, bool> {
-  bool operator () (const TrackerList::value_type& value) const { return value->is_usable(); }
+bool
+tracker_usable_t(const TrackerList::value_type& value) {
+  return value->is_usable();
 };
 
 bool
 TrackerList::has_usable() const {
-  return std::find_if(begin(), end(), tracker_usable_t()) != end();
+  return std::any_of(begin(), end(), tracker_usable_t);
```

Comment:
Simple functions like this should be using lambda functions.

Reply:
Will do. I avoided doing so as to not change it too much.